src/config.h.in

```diff
@@ -8,12 +8,23 @@
 #endif
 //#include "config-paths.h"
 
+/* Macro magic that expands a defined but empty variable to 1.
+ * `VAL` must be empty or a non-negative integer. */
+#define DO_ISEMPTY(VAL)  1 ## VAL
+#define ISEMPTY(VAL)     DO_ISEMPTY(VAL)
+
+
 /* dlite version */
 #define dlite_VERSION "@dlite_VERSION@"
 #define dlite_VERSION_MAJOR @dlite_VERSION_MAJOR@
 #define dlite_VERSION_MINOR @dlite_VERSION_MINOR@
 #define dlite_VERSION_PATCH @dlite_VERSION_PATCH@
 
+/* Require major and patch version to be defined */
```

Comment:
This comment does not reflect the following code. I think this will do the trick, but would it not be smart to test that all constants are set for completeness? In case something changes in cmake down the line you get an understandable error message

Reply:
Fixed

Reply:
Good point. It is better to be explicit about our requirement than relying on that the current way the version is provided in the cmake project() command makes it impossible to provide the patch without also providing the major and minor components as well.

Updated the comment and test accordingly.